app/controllers/works_controller.rb

```diff
@@ -32,6 +34,11 @@ def show
 
   def work_params
     params.require(:work).permit(:title, :work_type, :subtype, :contact_email,
+                                 'published(1i)', 'published(2i)', 'published(3i)',
```

Comment:
Can you say more about this `'attr({number}i)'` pattern used in this PR? It's new to me. Is it a reform incantation? Something something virtual attributes? I wonder if this is new/different enough to the team that it's worth adding a comment?

Reply:
That's how rails sends dates that are in parts: https://apidock.com/rails/ActionView/Helpers/DateHelper/date_select#422-Date-select-with-assert-valid-keys

Reply:
@jcoyne :beers: thank you. Worth adding a comment about that? It was new to me, anyway.